src/main/java/uk/gov/ons/ctp/response/action/service/impl/ActionRequestValidator.java

```diff
@@ -17,69 +16,17 @@
 
     /**
      * Validates whether the ActionRequest should be sent to a handler service.
-     * Contains the business logic for deciding whether the recipient is to receive an email or letter,
-     * dependent on the status of their account and the response.
-     * @param actionType
-     * @param actionRequest
```

Comment:
We should keep `@param actionRequest` as we're still passing it in

Reply:
It cries on codacy expecting a param tag in the function if this is there, + it's pretty self explanatory. If we actually ever generate java docs I'll put it back.